relay-server/src/actors/upstream.rs

```diff
@@ -1394,14 +1424,17 @@ impl UpstreamBroker {
 
         match status {
             RequestOutcome::Dropped => self.conn.notify_error(&self.action_tx),
-            RequestOutcome::Received => self.conn.reset_error(),
+            RequestOutcome::Received => {
+                self.conn.reset_error();
+                self.queue.retry_backoff_reset();
```

Comment:
```suggestion
                self.queue.reset_retry_backoff();
```

Personally I find it more clear to lead with 'reset' if it's a function that resets something, like in the `reset_error` function, what do you think ?

Reply:
I agree! I prefer (starting with) verbs in function names.